release/ray_release/glue.py

```diff
@@ -570,7 +570,10 @@ def run_release_test(
 
     if pipeline_exception:
         buildkite_group(":rotating_light: Handling errors")
-        exit_code, buildkite_exit_code, runtime = handle_exception(pipeline_exception)
+        exit_code, buildkite_exit_code, runtime = handle_exception(
```

Comment:
nit: "runtime" is quite confusing.. maybe "time_duration"?

Reply:
100% agree that the name is confusing ;), but it's already tied to a column name in databrick table: runtime -> result.runtime -> release_test_result.runtime, so I probably keep the name for consistency?